core/src/test/java/org/frankframework/jdbc/StoredProcedureQuerySenderTest.java

```diff
@@ -147,13 +184,15 @@ public void testSimpleStoredProcedureResultQueryInputParameters() throws Excepti
 		assertEquals("1", result.getResult().asString());
 	}
 
-	@Test
-	public void testSimpleStoredProcedureBlobInputParameter() throws Exception {
-		assumeThat("H2, PSQL, DB2 not supported for this test case", productKey, is(not(oneOf("H2", "PostgreSQL", "DB2"))));
+	@DatabaseTest
+	public void testSimpleStoredProcedureBlobInputParameter(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		assumeFalse(dataSourceName.contains("H2"), "H2, PSQL, DB2 not supported for this test case");
+		assumeFalse(dataSourceName.contains("PostgreSQL"), "H2, PSQL, DB2 not supported for this test case");
+		assumeFalse(dataSourceName.contains("DB2"), "H2, PSQL, DB2 not supported for this test case");
```

Comment:
Please try to write this as a single statement somehow? This is ugly.
Perhaps you can write one or two helper-methods to validate the datasource-name against supported (or the not-supported) databases so you can put them all in a varargs parameter array.